stablehlo/tests/print_types_invalid.mlir

```diff
@@ -85,3 +85,59 @@ func.func @one_result_type(%arg0: tensor<1xf64>) -> tensor<1xf64> {
   %0 = stablehlo.abs %arg0 : %arg0
   func.return %0 : tensor<1xf64>
 }
+
+// -----
+
+func.func @reduce_precision_not_literal(%arg0: tensor<3x4xf32>) -> (tensor<3x4xf32>) {
+  // expected-error @+1 {{custom op 'stablehlo.reduce_precision' expected valid keyword}}
+  %0 = stablehlo.reduce_precision %arg0, format = "e2m2" : tensor<3x4xf32>
+  func.return %0 : tensor<?x?xf64>
+}
+
+// -----
+
+func.func @reduce_precision_no_em(%arg0: tensor<3x4xf32>) -> (tensor<3x4xf32>) {
+  // expected-error @+1 {{custom op 'stablehlo.reduce_precision' expected exponent mantissa in format e#m#, saw z4f2}}
```

Comment:
As I'm looking through StablehloOps.cpp, I see that we have a bunch of different ways to express expected vs actual mismatches: "expected, saw", "expected, got", "expected, found", etc. We'll need to sort this out at some later point.

Reply:
Agree. Created #63.